src/QsCompiler/Tests.Compiler/ExecutionTests.fs

```diff
@@ -139,7 +139,7 @@ type ExecutionTests (output:ITestOutputHelper) =
             "
 
 
-    [<Fact>]
+    // TODO: [<Fact>]
```

Comment:
This should be [<Fact(Skip="Reason")>]. This will make the skipped test more visible and less likely to be missed in the future.